packages/firestore-types/index.d.ts

```diff
@@ -216,6 +216,13 @@ export class FirebaseFirestore {
    */
   app: any;
 
+  /**
+   * Clears the persistence cache. This can only be called when the client is not running.
+   *
+   * @return A promise that is resolved once the persistence has been cleared.
+   */
+  clearPersistence(): Promise<void>;
```

Comment:
FYI: This change also needs to be made in packages/firebase/index.d.ts :(

Reply:
Unfortunately, you should hold both changes back for a separate PR.

These changes essentially make the API public and we need to wait until the API has been approved before we can do that. We can commit the implementation though, and that's worth doing so this isn't sitting around in a branch and rotting while we work through approval.